r-package/R/dictionary.R

```diff
@@ -0,0 +1,57 @@
+#'
+#'
+#' Open the data set's dictionary.
+#'
+#' @param dataset a string containing the name of a basedosdados' data set, the dictionary of which you are asking for.
+#' @param billing_project_id a string containing your billing project id. If you've run `set_billing_id` then feel free to leave this empty.
```

Comment:
```suggestion
#'
```

Não usamos mais o parâmetro `page_size`, a API do `bigrquery` melhorou muito nesse sentido e escolhe para a gente.

Reply:
mas po esse é o parametro `billing_project_id`, esse tem que ta de qualquer jeito nao?

Reply:
Erro meu, selecionei a linha errada hahaha